lib/irb/extend-command.rb

```diff
@@ -255,33 +255,12 @@ def self.def_extend_command(cmd_name, cmd_class, load_file = nil, *aliases)
         cmd_class = cmd_class.name
       end
 
-      if load_file
```

Comment:
Since we assume load_file to be always present, I think we can change optional arg `load_file` to required.
```diff
- def self.def_extend_command(cmd_name, cmd_class, load_file = nil, *aliases)
+ def self.def_extend_command(cmd_name, cmd_class, load_file, *aliases)
```

Reply:
Good catch!